tests/cpu_bugs/xen_domu_mitigation_test.pm

```diff
@@ -703,69 +703,94 @@ sub install_domu {
               . " --serial pty", timeout => 5400);
     }
     # Check if the DomU is up
-    script_run("virsh start \"${domu_name}\"");
+    my $vmstatus = script_output("virsh start \"${domu_name}\"");
```

Comment:
?? This check point is deceiving yourself, this checking did not sure if the guest is up or usable.